concreep-refilled/control.lua

```diff
@@ -55,7 +55,7 @@ function init()
 
 	global.creepers = {}
 	global.creeper_index = 1
-	global.robots_given_instructions = 0
+	global.robots_given_instructions = 0 --not used atm - what was the intention here?

```

Comment:
was planning to remove this after merging your PR

Reply:
Well - leaving it up to you in that case - maybe we have some idea till then